pydatastructs/linear_data_structures/algorithms.py

```diff
@@ -1369,3 +1372,58 @@ def bubble_sort(array, **kwargs):
         array._modify(force=True)
 
     return array
+
+def linear_search(array, elem):
+    for i in range(array._size):
+        if array._data[i] == elem:
+            return i
+    return -1
+
+def binary_search(array, elem, comp = None):
+    L = 0
+    R = array._size - 1
```

Comment:
`left`, `right`?

Reply:
Yes

Reply:
Please make it explicit then. Always use full names instead of short hand notations. Not many key strokes will be saved while typing, `L` instead of `left`.